webdriver-spec.html

```diff
@@ -4999,7 +4999,56 @@
     <p class=issue>Ensure all correct events are fired.
 
    <dt>Otherwise
-   <dd><p class=issue>Run interaction steps on coordinate.
+   <dd>
+    <ol>
+     <li><p>Let <var>mouse</var> be a new <a>pointer input source</a>.
+
+     <li><p>Let <var>click point</var> be
+      the <var>element</var>’s <a>in-view center point</a>.
+
+     <li><p>Let <var>pointer move action</var> be an <a>action object</a>
+      with the <var>mouse</var>’s <code>id</code>, "<code>mouse</code>",
+      and "<code>pointerMove</code>" as arguments.
+
+     <li><p>Set <var>pointer move action</var>’s
+      <code>x</code> and <code>y</code> properties
+      to those of <var>click point</var>.
+
+     <li><p>Set <var>pointer move action</var>’s
+      <code>origin</code> property to "<code>viewport</code>".
```

Comment:
Aren't `click point` and the centre of the element the same thing. Might be better to use an `origin` of `element` and `x` and `y` of `0,0`